src/LogglyLoggerFormatter.ts

```diff
@@ -32,21 +21,21 @@ export class LogglyLoggerFormatter {
         // "timezone": "UTC"
         */
       },
-    };
+    }
 
     if (EnvInspect.isWeb()) {
       json.url_info = {
         host: window.location.hostname,
         path: window.location.pathname,
         query: window.location.search,
         url: window.location.href,
-      };
+      }
       if (document.referrer) {
-        json.url_info.referrer = document.referrer;
+        json.url_info.referrer = document.referrer
       }
```

Comment:
I believe document is not defined in SSR too?

Reply:
this block is under `isWeb` check